tests/test_lstm.py

```diff
@@ -51,6 +51,7 @@ def new_graph_validator(g):
                 # Skip checks for tflite graphs (no ":" in outputs)
                 return good
             good = good and check_op_count(g, "LSTM", require_lstm_count, disabled=False)
+            # if LSTM op rewriter matchs failed, Loop op will be shown generally.
```

Comment:
Let's change it to:

If LSTM op rewriter failed to work, Loop op will be shown in general.